net/bmx7/Makefile

```diff
@@ -0,0 +1,162 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bmx7
+PKG_VERSION:=cc245a22e54b5e110384485f18d9b805c91d8b71
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/bmx-routing/bmx7/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=bc68f2ce8ff9c435deff31833bdb39eafa3aeda1c19809705e2b4b1f1d4c8256
+
+PKG_MAINTAINER:=Axel Neumann <neumann@cgws.de>
+PKG_LICENSE:=GPL-2.0
```

Comment:
This should be -or-later
PKG_LICENSE_FILES should be LICENSE

Reply:
Is that actually a requirement?

Reply:
We want consistency between packages. From a quick look, this is -or-later.

Reply:
Put another way, the unofficial standard for PKG_LICENSE is SPDX tags. See: https://spdx.org/licenses/

Reply:
updated, thanks